profiles/steve-booth/index.md

```diff
@@ -9,4 +9,4 @@ twitter_username: blank
 linkedin_username: blank
 ---
 
-Steve Booth is a seasoned sales executive with over 30 years of enterprise solution sales and sales management experience. Steve is passionate about the opportunities for blockchain software to truly change the world for the better and is focused on Chia blockchain adoption across the globe.  Prior to joining Chia, he served as vice president of sales at Vindicia. Before that, he was vice president of sales and business development at Proclivity Systems. Steve has also held senior sales management positions at Aria Systems, QlikTech, Responsys, Portal Software, Oracle, and ADP. He holds a BS in marketing and finance from LaSalle University.
\ No newline at end of file
+Steve Booth is a seasoned sales executive with over 30 years of enterprise solution sales and sales management experience. Steve is passionate about the opportunities for distributed ledger technology to truly change the world for the better, and is focused on Chia blockchain adoption across the globe.  Prior to joining Chia, he served as vice president of sales at Vindicia. Before that, he was vice president of sales and business development at Proclivity Systems. Steve has also held senior sales management positions at Aria Systems, QlikTech, Responsys, Portal Software, Oracle, and ADP. He holds a BS in marketing and finance from LaSalle University.
```

Comment:
I loathe DLT. It's either blockchain or secure distributed database.

Reply:
Point taken :-) I don't like it either but it's used in many business publications.